### PR TITLE
Show emoji corresponding to coverage value

### PR DESCRIPTION
**Pull request by mingrammer**
(#24) Show emoji corresponding to coverage value
1. Picked the moon emoji to represent progress ratio.
2. Ratio range has been expanded.

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+def get_progress_emoji(ratio):
```
Comment (hunkim) [inline utils.py:1]: Can we us a size 11 list?
emoji = [':new_moon:', ':new_moon:', ':waning_crescent_moon:', ':waning_crescent_moon:', ...,':full_moon:']

Then, you can simply return emoji [(int)ration/10]. What do you think?

File: sample.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 # Translation Coverage                         
 (Automatically generated. DO NOT edit.)
-* [/](/) (27/55) [49%]
-  * [/sample.md](/sample.md) (12/29) [41%]
-  * [**/sample_eng.md**](/sample_eng.md) (0/11) [0%]
-  * [*/sample_kor.md*](/sample_kor.md) (15/15) [100%]
+* :last_quarter_moon: [/](/) (27/55) [49%]
```
Comment (hunkim) [inline sample.md:3]: Looks very good!

**Comment by hunkim** [comment]
Looks very good. I have only one minor comment. Thanks!

File: utils.py (hunk 1 above)
Comment (mingrammer) [reply utils.py:1]: If there are 10 levels, that is good choice.
But, we have just 5 levels, so the list may have duplicated items up to 4. it seems not clean.

Comment (hunkim) [reply utils.py:1]: I know but the code is very efficient. :-) Also you can try rate/20.

Comment (mingrammer) [reply utils.py:1]: ratio/20 is better. I'll apply

Comment (mingrammer) [reply utils.py:1]: How about 20.2? There are 101 integers.

Comment (hunkim) [reply utils.py:1]: sure.

Comment (mingrammer) [reply utils.py:1]: Review plz.

**Comment by hunkim** [comment]
Cooool!

**Comment by hunkim** [comment]
I know but the code is very efficient. :-)

**Comment by hunkim** [comment]
Or rate/20. :-)

**Pull request merged**